bundles/specmate-jira-connector/src/com/specmate/connectors/jira/internal/services/JiraConnector.java

```diff
@@ -341,8 +345,36 @@ public IContainer getContainerForRequirement(Requirement requirement) throws Spe
 	}
 
 	@Override
-	public boolean authenticate(String username, String password) throws SpecmateException {
-		return JiraUtil.authenticate(url, projectName, username, password);
+	public Set<IProject> authenticate(String username, String password, IProject logonProject, IProjectService projectService) throws SpecmateException {		
+		
+		List<String> accessibleJiraProjectNames = JiraUtil.getProjects(url, username, password);
+		Set<IProject> accessibleSpecmateProjectNames = new HashSet<>();
+		
+		if (! (logonProject.getConnector() instanceof JiraConnector)) {

```

Comment:
as discussed: remove first part of the method and eliminate logonProject and projectService parameter

Reply:
dome